src/SourceGenerators/Uno.UI.Tasks/BatchMergeXaml/CustomTask.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Uno.UI.Tasks.BatchMerge
+{
+    public abstract class CustomTask : Microsoft.Build.Utilities.Task
+    {
+        protected bool HasLoggedErrors
+        {
+            get
+            {
+                if (BuildEngine != null)
+                {
+                    return Log.HasLoggedErrors;
+                }
+                else
+                {
+                    return hasLoggedErrors;
+                }
+            }
+        }
+
+        private bool hasLoggedErrors = false;
+
+        protected void LogMessage(string message)
+        {
+            if (BuildEngine != null)
+            {
+                Log.LogMessage(message);
+            }
+            else
+            {
+                Console.WriteLine(message);
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5930374)